### DISCUSSION TITLE
any npm library for angular js? pls update. Thanks in advance!

### DISCUSSION THREAD

**Discussion by mt-vijayakumar-m** [pikasojs/pikaso#91] [Q&A]
Hi Team,

any npm library for angular js? pls update. Thanks in advance!
if not how to integrate into angular app?
because Konva JS has npm library for angular .

Regards,
VJ

**Comment by raminious** [comment]
In general, it should work with any library, not just Angular. Angular is a library I am unfamiliar with, so I don't have a working example for it. If you do this and provide an example for others, I appreciate it.

**Reply by mt-vijayakumar-m** [reply]
Thank you @raminious  for your time and updates :)

**Reply by raminious** [reply]
anytime 👍
